canvas_modules/harness/test_resources/parameterDefs/slider_paramDef.json

```diff
@@ -0,0 +1,169 @@
+{
+  "titleDefinition": {
+    "title": "Slider with Min Max"
+  },
+  "current_parameters": {
+    "slider": 50,
+    "slider_error": 50,
+    "hide": true,
+    "slider-disable": false
+  },
+  "parameters": [
+    {
+      "id": "slider-disable",
+      "type": "boolean"
+	  },
+    {
+      "id": "slider",
+      "type": "integer"
+    },
+    {
+      "id": "slider_error",
+      "type": "object",
```

Comment:
This shouldn't be an object

Reply:
can we also test with double values.  Something with 0 and 1 being the min/max with increments of 0.1